### PR TITLE
[EDU-3624] refactor: update Bot Protection guide

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-3624]

### Changes

- Updating Bot Protection guide (EN and PT) to include new features.

### Additional links

https://www.azion.com/en/documentation/products/guides/bot-protection/

[EDU-3624]: https://aziontech.atlassian.net/browse/EDU-3624?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 1]
```diff
@@ -5,37 +5,40 @@ meta_tags: bot, protection, cybersecurity, edge computing
...
-To use Azion **Bot Protection**, access [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) and go to **Marketplace**.
+To use Azion **Bot Protection**:
 
-On the Marketplace's homepage, select the integration's card.
+1. Access [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) > **Marketplace**.
+2. On the Marketplace's homepage, select the integration's card.
+3. Once the integration's page opens, click the **Get It Now** button, at the bottom-right corner of the page.
 
-Once the integration's page opens, click the **Get It Now** button, at the bottom-right corner of the page, and a successful message appears to confirm your integration is installed.
+A successful message appears to confirm your integration is installed.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:31]:
```suggestion
A successful message appears to confirm that your integration has been installed.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 2]
```diff
@@ -99,27 +104,71 @@ Where:
...
+- `deny`: delivers a standard *Status Code 403* response.
+- `drop`: terminates the request without a response to the user.
+- `redirect`: allows the request to be redirected to a new URL/Location when the security threshold is reached. To enable this action, you must declare the variables as in the example:
+
+```json
+  "action": "redirect",
+  "redirect_to": "http://xxxxxxxxxx.map.azionedge.net/"
+```
+
+Where `redirect_to`defines the new URL/Location to redirect the requests. If this field isn't filled or it's filled with a value that isn't an *string*, the function will behave as would happen with the `allow` action enabled.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:145]:
```suggestion
Where `redirect_to` defines the new URL/Location to redirect the requests. If this field isn't filled or it's filled with a value that isn't a *string*, the function will behave as if the `allow` action were enabled.
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 3]
```diff
@@ -99,27 +104,71 @@ Onde:
...
+- `deny`: entrega uma resposta padrão com *Status Code 403*.
+- `drop`: encerra a requisição sem uma resposta ao usuário.
+- `redirect`: permite que a requisição seja redirecionada para uma nova URL/Localização quando o limite de segurança é atingido. Para habilitar esta ação, você deve declarar as variáveis como no exemplo:
+
+```json
+  "action": "redirect",
+  "redirect_to": "http://xxxxxxxxxx.map.azionedge.net/"
+```
+
+Onde `redirect_to` define a nova URL/Localização para redirecionar as requisições. Se este campo não estiver preenchido ou estiver preenchido com um valor que não seja uma string, a função se comportará como aconteceria com a ação `allow` habilitada.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:145]:
```suggestion
Onde:

- `redirect_to` define a nova URL/Localização para redirecionar as requisições. Se este campo não estiver preenchido ou estiver preenchido com um valor que não seja uma string, a função se comportará como aconteceria com a ação `allow` habilitada.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 4]
```diff
@@ -70,7 +75,7 @@ This action will load the function, showing a form with the function code and, j
 }
 ```
 
-Even when `threshold` and `action` are mandatory variables to be defined, you can add and define more variables, according to your needs, as showed in the example below that includes [integration with a SIEM](#integrating-with-a-siem).
+Even when `threshold` and `action` are mandatory variables, you can add and define more variables, according to your needs, as showed in the example below that includes [integration with a SIEM](#integrating-with-a-siem).
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:78]:
```suggestion
Even when `threshold` and `action` are mandatory variables, you can add and define more variables, according to your needs, as shown in the example below that includes [integration with a SIEM](#integrating-with-a-siem).
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 5]
```diff
@@ -99,27 +104,73 @@ Where:
...
 | `reputation_network_lists` | Array of numbers | No | Every network list that should be used to validate the request IP. If the request IP is found in a list, then the request score will be increased. If the request IP is found in multiple lists, the score will be increased by multiple times |
 | `session_signature_key` | String | No | To sign the `az_asm`. If this field has no value or an invalid value, the function will use the default value `azion` |
 | `siem_integration` | Boolean | No | Defines if the request data should perform POST to an external endpoint. Default value: `false` |
-| `send_data_to_siem_when` | String | No | Defines when the request data should be sent to the external endpoint (if `siem_integration` is enabled). Possible values: `always`, 'deny', `drop`, or `log` (to send the data to the SIEM endpoint, if integrated) |
+| `send_data_to_siem_when` | String | No | Defines when the request data should be sent to the external endpoint (if `siem_integration` is enabled). Possible values: `always`, 'deny', `redirect`, `custom_html`, `drop`, or `log`. Read more about [configuring actions](#configuring-actions) |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:114]:
```suggestion
| `send_data_to_siem_when` | String | No | Defines when the request data should be sent to the external endpoint (if `siem_integration` is enabled). Possible values: `always`, `deny`, `redirect`, `custom_html`, `drop`, or `log`. Read more about [configuring actions](#configuring-actions) |
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 6]
```diff
@@ -99,27 +104,73 @@ Where:
...
 | `siem_connection_args.uri` | String | No | The URI where the data will be POSTed to. Default value: `/`  |
 | `siem_connection_args.port` | Number | No | The port where the data will be POSTed to. Default value: `443` |
 | `siem_connection_args.scheme` | String | No | Defines the scheme to be used in the POST. Default value: `https` |
 | `siem_connection_args.additional_headers` | Table | No | Additional headers to be sent in the POST. The function will always send the following headers: `["Accept"] = "application/json"`, `["Content-Type"] = "application/json"` |
+| `mode`| String | No | The defined environment to run the fuction. Possible values: `api` or `web`. Read more about [Bot Protection modes](#bot-protection-modes) |
+| `user_id_source`| String | No | The source to extract the data to create the user ID for the request. Possible values (case-sensitive in lowercase): `args` (querystring), `cookie`, `header`, or `body`(request body). **Note**: only bodies in the `application/json`, `application/x-www-form-urlencoded`, and `multipart/form-data` formats are accepted |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:122]:
```suggestion
| `user_id_source`| String | No | The source to extract the data to create the user ID for the request. Possible values (case-sensitive in lowercase): `args` (query string), `cookie`, `header`, or `body`(request body). **Note**: only bodies in the `application/json`, `application/x-www-form-urlencoded`, and `multipart/form-data` formats are accepted |
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 7]
```diff
@@ -99,27 +104,73 @@ Where:
...
-When you're done, click the **Save** button.
+5. When you're done, click the **Save** button.
 
 :::tip
 To learn how to get the logs of Azion **Bot Protection**, go to the [Integrating with a SIEM](#integrating-with-a-siem) section.
 :::
 
+#### Configuring actions
+
+Azion **Bot Protection** is able to execute **5 different actions** whenever the request's score is greater or equals the defined threshold:
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:133]:
```suggestion
Azion **Bot Protection** is able to execute **5 different actions** whenever the request's score is greater than or equals the defined threshold:
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 8]
```diff
@@ -99,27 +104,73 @@ Where:
...
+
+- `custom_html`: allows customized HTML content to be delivered to the user in case of a threshold violation. To enable this action, you must declare the variables as in the example:
+
+```json
+  "action": "custom_html",
+  "custom_html": "This should be the custom HTML content",
+  "custom_status_code": 418,
+```
+
+Where `custom_html` defines the HTML content to be delivered and `custom_status_code` the Status Code to be delivered.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:157]:
```suggestion
Where `custom_html` defines the HTML content to be delivered and `custom_status_code` the status code to be delivered.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 9]
```diff
@@ -99,27 +104,73 @@ Where:
...
+
+- If `custom_html` isn't filled or it's filled with a value that isn't an *string*, the function will behave as would happen with the `allow` action enabled.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:159]:
```suggestion
- If `custom_html` isn't filled or it's filled with a value that isn't a *string*, the function will behave as would happen with the `allow` action enabled.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 10]
```diff
@@ -99,27 +104,73 @@ Where:
...
+- If `custom_status_code` isn't filled or it's filled with a value that isn't a *number*, the default value will be a *Status Code 200*.
+
+#### Bot Protection modes
+
+Azion **Bot Protection** allows you to define the environment in which the function is expected to run, being API or an web application the possible modes.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:164]:
```suggestion
Azion **Bot Protection** allows you to define the environment in which the function is expected to run. The possible modes are *API* or *web application*.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 11]
```diff
@@ -196,13 +247,16 @@ Azion will provide you with easy-to-go configurations, that should be enough for
 
 To find this file:
 
-1. On the RTM homepage, select the **Products menu**, indicated by the three horizontal lines on the upper-left corner.
-2. Under the **SECURE** section, select **Edge Firewall**.
-3. Locate the **Azion Bot Protection** edge function and click on it to load the integration's source-code form.
-4. Select the **Arguments** tab.
+1. On the upper-left corner, select **Products menu** > **Edge Firewall** in the **SECURE** section.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:250]: passo 2 não deveria ser selecionar a aba Functions?

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 12]
```diff
@@ -196,13 +247,16 @@ Azion will provide you with easy-to-go configurations, that should be enough for
...
+2. Locate the **Azion Bot Protection** edge function and click on it to load the integration's source-code form.
+3. Select the **Arguments** tab.
 
 This will load a `JSON` file where you'll be able to tune **Azion Bot Protection** according to the necessities of your business.
 
+<Button href="/en/documentation/products/guides/secure/manage-bots/" text="go to manage bots guide" variant="secondary">
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:256]: checar se vai subir a tempo

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 13]
```diff
@@ -5,27 +5,30 @@ meta_tags: bot, protection, cibersegurança, edge computing, SIEM
...
+
 ---
 
 ## Obtenha a integração
 
-Para obter a integração, acesse no [Real-Time Manager (RTM)](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) e vá para o **Marketplace**.
+Para obter a integração:
 
-Na página inicial do Marketplace, selecione o card correspondente à integração Azion **Bot Manager**.
+1. Acesse no [Real-Time Manager (RTM)](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) > **Marketplace**.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:27]:
```suggestion
1. Acesse o [Real-Time Manager (RTM)](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) > **Marketplace**.
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 14]
```diff
@@ -61,7 +64,9 @@ Ainda na página do **Edge Firewall**:
 3. Dê um nome fácil de lembrar à sua instância.
 4. No menu suspenso, selecione a função **Azion Bot Protection**.
 
-Esta ação carregará a função, mostrando um formulário com o código-fonte desta e, logo acima dele, duas abas: **Code** e **Arguments**. Na aba **Arguments**, você passará suas variáveis.
+Esta ação carregará a função, mostrando um formulário com o código-fonte desta e, logo acima dele, duas abas: **Code** e **Ars**.
+
+Na aba **Ars**, você passará suas variáveis.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:69]:
```suggestion
Na aba **Args**, você passará suas variáveis.
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 15]
```diff
@@ -99,27 +104,73 @@ Onde:
 | Variável | Tipo | Obrigatório | Descrição |
 |---|---|---|---|
 | `threshold` | Número | Sim | A pontuação máxima que a requisição pode atingir antes que a função execute uma ação. Se não tiver valor, a função não agirá |
-| `action` | String | Sim | A ação a ser tomada pela função sempre que a pontuação da requisição for igual ou maior ao limite definido. Valores possíveis: `deny`, `drop` e `log` (para enviar os dados ao endpoint do SIEM, se integrado) |
+| `action` | String | Sim | A ação a ser tomada pela função sempre que a pontuação da requisição for igual ou maior ao limite definido. Valores possíveis: `deny`, `redirect`, `custom_html`, `drop` e `log`. Saiba mais sobre a [configuração de actions](#configuração-de-actions) |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:107]:
```suggestion
| `action` | String | Sim | A ação a ser tomada pela função sempre que a pontuação da requisição for igual ou maior ao limite definido. Valores possíveis: `deny`, `redirect`, `custom_html`, `drop` e `log`. Saiba mais sobre a [configuração de actions](#configuracao-de-actions) |
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 16]
```diff
@@ -99,27 +104,73 @@ Onde:
...
 | `session_signature_key` | String | Não | Para assinar o `az_asm`. Se este campo não tiver valor ou tiver um valor inválido, a função usará o valor padrão `azion` |
 | `siem_integration` | Boolean | Não | Define se os dados da requisição devem ser postados em um endpoint externo. Valor padrão: `false` |
-| `send_data_to_siem_when` | String | Não | Define quando os dados da solicitação devem ser enviados ao endpoint externo (se `siem_integration` estiver habilitado). Valores possíveis: `always`, ‘deny`, `drop` ou `log` (para enviar os dados para o endpoint do SIEM, se integrado) |
+| `send_data_to_siem_when` | String | Não | Define quando os dados da solicitação devem ser enviados ao endpoint externo (se `siem_integration` estiver habilitado). Valores possíveis: `always`, `deny`, `redirect`, `custom_html`, `drop` e `log`. Saiba mais sobre a [configuração de actions](#configuração-de-actions) |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:114]:
```suggestion
| `send_data_to_siem_when` | String | Não | Define quando os dados da solicitação devem ser enviados ao endpoint externo (se `siem_integration` estiver habilitado). Valores possíveis: `always`, `deny`, `redirect`, `custom_html`, `drop` e `log`. Saiba mais sobre a [configuração de actions](#configuraca-de-actions) |
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 17]
```diff
@@ -99,27 +104,73 @@ Onde:
...
 | `siem_connection_args.uri` | String | Não | A URI onde os dados serão postados. Valor padrão: `/` |
 | `siem_connection_args.port` | Número | Não | A porta para onde os dados serão POSTados. Valor padrão: `443` |
 | `siem_connection_args.scheme` | String | Não | Define o esquema a ser utilizado no POST. Valor padrão: `https` |
 | `siem_connection_args.additional_headers` | Tabela | Não | Cabeçalhos adicionais a serem enviados no POST. A função sempre enviará os seguintes cabeçalhos: `["Accept"] = "application/json"`, `["Content-Type"] = "application/json"` |
+| `mode` | String | Não | O ambiente definido para executar a função. Valores possíveis: `api` e `web`. Saiba mais sobre [Bot Protection modes](#bot-protection-modes) |
+| `user_id_source` | String | Não | A fonte onde serão extraídos os dados para criar o user ID da requisição. Valores possíveis(case-sensitive em minúsculas): `args` (querystring), `cookie`, `header` ou `body`(body da requisição). **Nota**: apenas os bodies nos formatos `application/json`, `application/x-www-form-urlencoded`, and `multipart/form-data` são aceitos  |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:122]:
```suggestion
| `user_id_source` | String | Não | A fonte onde serão extraídos os dados para criar o user ID da requisição. Valores possíveis(case-sensitive em minúsculas): `args` (query string), `cookie`, `header` ou `body`(body da requisição). **Nota**: apenas os bodies nos formatos `application/json`, `application/x-www-form-urlencoded` e `multipart/form-data` são aceitos  |
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 18]
```diff
@@ -99,27 +104,73 @@ Onde:
...
 :::
 
+#### Configuração de actions
+
+Azion **Bot Protection** é capaz de executar 5 ações diferentes sempre que a pontuação da requisição for maior ou igual ao limite definido:
+
+- `allow`: permite a continuação da solicitação.
+- `deny`: entrega uma resposta padrão com *Status Code 403*.
+- `drop`: encerra a requisição sem uma resposta ao usuário.
+- `redirect`: permite que a requisição seja redirecionada para uma nova URL/Localização quando o limite de segurança é atingido. Para habilitar esta ação, você deve declarar as variáveis como no exemplo:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:138]: Localização precisa ser uppercase?

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 19]
```diff
@@ -99,27 +104,73 @@ Onde:
...
+
+- `custom_html`: permite a entrega de conteúdo HTML personalizado ao usuário em caso de violação do limite. Para habilitar esta ação, você deve declarar as variáveis como no exemplo:
+
+```json
+  "action": "custom_html",
+  "custom_html": "This should be the custom HTML content",
+  "custom_status_code": 418,
+```
+
+Onde `custom_html` define o conteúdo HTML a ser entregue e `custom_status_code` o Status Code a ser entregue.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:157]:
```suggestion
Onde `custom_html` define o conteúdo HTML a ser entregue e `custom_status_code` define o status code a ser entregue.
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 20]
```diff
@@ -99,27 +104,73 @@ Onde:
...
+
+- Se `custom_html` não estiver preenchido ou estiver preenchido com um valor que não seja uma string, a função se comportará como aconteceria com a ação `allow` habilitada.
+- Se `custom_status_code` não estiver preenchido ou estiver preenchido com um valor que não seja um número, o valor padrão será um *Status Code 200*.
+
+#### Bot Protection modes
+
+Azion **Bot Protection** permite que você defina o ambiente em que a função deve ser executada, sendo API ou uma aplicação web os modos possíveis.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:164]:
```suggestion
Azion **Bot Protection** permite que você defina o ambiente em que a função deve ser executada. Os modos possíveis são *API* ou uma *aplicação web*.
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 21]
```diff
@@ -195,13 +246,15 @@ A Azion fornecerá configurações básicas e fáceis de usar, que devem ser suf
 
 Para encontrar esse arquivo:
 
-1. Na página inicial do RTM, selecione o **Products menu**, indicado pelas três linhas horizontais no canto superior esquerdo.
-2. Na seção **SECURE**, selecione **Edge Firewall**. 
-3. Localize a edge function da Azion **Bot Protection** e clique nela para carregar o formulário contendo o código-fonte da integração.
-4. Selecione a aba **Arguments**.
+1. No canto superior esquerdo do RTM, selecione **Products menu** > **Edge Firewall**, dentro da seção **SECURE**.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:249]: passo 2 a aba functions?

**Review by hannah-sk — APPROVED** [review]
deixei algumas sugestões e dúvidas

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 22]
```diff
@@ -196,13 +247,16 @@ Azion will provide you with easy-to-go configurations, that should be enough for
 
 To find this file:
 
-1. On the RTM homepage, select the **Products menu**, indicated by the three horizontal lines on the upper-left corner.
-2. Under the **SECURE** section, select **Edge Firewall**.
-3. Locate the **Azion Bot Protection** edge function and click on it to load the integration's source-code form.
-4. Select the **Arguments** tab.
+1. On the upper-left corner, select **Products menu** > **Edge Firewall** in the **SECURE** section.
+2. Locate the **Azion Bot Protection** edge function and click on it to load the integration's source-code form.
+3. Select the **Arguments** tab.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:252]:
```suggestion
2. Select the one related to **Bot Protection**.
3. Open the **Functions** tab to load the integration's source-code form.
3. Select the **Args** tab.
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 23]
```diff
@@ -195,13 +246,15 @@ A Azion fornecerá configurações básicas e fáceis de usar, que devem ser suf
...
+2. Localize a edge function da Azion **Bot Protection** e clique nela para carregar o formulário contendo o código-fonte da integração.
+3. Selecione a aba **Arguments**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:251]:
```suggestion
2. Selecione o relacionado com **Bot Protection**. 
3. Abra a aba **Functions** para carregar o formulário contendo o código-fonte da integração..
4. Selecione a aba **Args**.
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 24]
```diff
@@ -99,27 +104,73 @@ Onde:
...
 :::
 
+#### Configuração de actions
+
+Azion **Bot Protection** é capaz de executar 5 ações diferentes sempre que a pontuação da requisição for maior ou igual ao limite definido:
+
+- `allow`: permite a continuação da solicitação.
+- `deny`: entrega uma resposta padrão com *Status Code 403*.
+- `drop`: encerra a requisição sem uma resposta ao usuário.
+- `redirect`: permite que a requisição seja redirecionada para uma nova URL/Localização quando o limite de segurança é atingido. Para habilitar esta ação, você deve declarar as variáveis como no exemplo:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:138]:
```suggestion
- `redirect`: permite que a requisição seja redirecionada para uma nova URL/localização quando o limite de segurança é atingido. Para habilitar esta ação, você deve declarar as variáveis como no exemplo:
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx [hunk 25]
```diff
@@ -99,27 +104,73 @@ Where:
...
+- `redirect`: allows the request to be redirected to a new URL/Location when the security threshold is reached. To enable this action, you must declare the variables as in the example:
+
+```json
+  "action": "redirect",
+  "redirect_to": "http://xxxxxxxxxx.map.azionedge.net/"
+```
+
+Where:
+
+- `redirect_to`: defines the new URL/Location to redirect the requests. If this field isn't filled or it's filled with a value that isn't a *string*, the function will behave as if the `allow` action were enabled.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection/index.mdx:147]:
```suggestion
Where `redirect_to`: defines the new URL/location to redirect the requests. If this field isn't filled or filled with a value that isn't a *string*, the function will behave as if the `allow` action was enabled.
```

File: src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx [hunk 26]
```diff
@@ -99,27 +104,73 @@ Onde:
...
+
+```json
+  "action": "redirect",
+  "redirect_to": "http://xxxxxxxxxx.map.azionedge.net/"
+```
+
+Onde:
+
+- `redirect_to` define a nova URL/Localização para redirecionar as requisições. Se este campo não estiver preenchido ou estiver preenchido com um valor que não seja uma string, a função se comportará como aconteceria com a ação `allow` habilitada.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/bot-protection/index.mdx:147]:
```suggestion
Onde `redirect_to` define a nova URL/Localização para redirecionar as requisições. Se este campo não estiver preenchido ou estiver preenchido com um valor que não seja uma string, a função se comportará como aconteceria com a ação `allow` habilitada.
```

**Review by bru-andrade — APPROVED** [review]

**Pull request closed without merging**